toqito/state_props/is_abs_ppt.py

```diff
@@ -0,0 +1,128 @@
+"""Check if a quantum state is absolutely PPT."""
+
+import numpy as np
+
+from toqito.matrix_props.is_positive_semidefinite import is_positive_semidefinite
+from toqito.state_opt.symmetric_extension_hierarchy import symmetric_extension_hierarchy
+from toqito.state_props.is_ppt import is_ppt
+from toqito.state_props.is_separable import is_separable
+
+
+def is_abs_ppt(rho: np.ndarray, dim: list[int] = None, max_constraints: int = 2612) -> int:
+    r"""Determine whether a quantum state is absolutely PPT.
+
+    A quantum state :math:`\rho` is absolutely PPT if it remains PPT under any unitary transformation.
+    This function evaluates the condition using:
+
+    - Quick separability checks (to immediately identify separable states)
+    - The PPT property (to test for the PPT condition)
+    - A Gershgorin-type condition (providing a bound based on the eigenvalues)
+    - A symmetric extension hierarchy (for deeper verification when needed)
+
+    .. math::
+        \rho \text{ is absolutely PPT} \iff \forall U,\, (U \rho U^\dagger)^{T_B} \succeq 0.
+
+    Examples
+    ========
+    Example 1: A separable mixed state (trivially PPT)
+
+    >>> import numpy as np
+    >>> from toqito.state_props import is_abs_ppt
+    >>> rho = np.array([[0.5, 0], [0, 0.5]])  # Maximally mixed state
+    >>> is_abs_ppt(rho, [2, 2])
+    1

```

Comment:
Assuming that the type-signature comment above is changed, we would need to convert these examples as well.